csharp/src/Microsoft.ML.OnnxRuntime/OnnxRuntime.cs

```diff
@@ -108,6 +108,17 @@ public void DisableTelemetryEvents()
             NativeApiStatus.VerifySuccess(NativeMethods.OrtDisableTelemetryEvents(Handle));
         }
 
+        /// <summary>
+        /// Create and register an arena based allocator to the OrtEnv instance
```

Comment:
```suggestion
        /// Create and register an allocator to the OrtEnv instance
```

Reply:
But really this method only supports registering an arena based allocator right ? So doesn't the comment convey that ?

Reply:
I just realized that we can register non-arena based allocators as well. The `OrtArenaCfg` parameter (which applied only to arena based allocators) confused me for a moment. But it can be null and the type of the allocator is defined by the OrtMemoryInfo instance sent in